// ews-javascript-api/js/Core/Responses/SyncResponse.js
"use strict";
var __extends = (this && this.__extends) || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
};
var ServiceResponse_1 = require("./ServiceResponse");
/**
 * ## *Not Implemented*
 */
var SyncResponse = (function (_super) {
    __extends(SyncResponse, _super);
    function SyncResponse() {
        _super.apply(this, arguments);
    }
    SyncResponse.prototype.CreateChangeInstance = function () { throw new Error("SyncResponse.ts - CreateChangeInstance : Not implemented."); };
    SyncResponse.prototype.GetChangeElementName = function () { throw new Error("SyncResponse.ts - GetChangeElementName : Not implemented."); };
    SyncResponse.prototype.GetChangeIdElementName = function () { throw new Error("SyncResponse.ts - GetChangeIdElementName : Not implemented."); };
    SyncResponse.prototype.GetIncludesLastInRangeXmlElementName = function () { throw new Error("SyncResponse.ts - GetIncludesLastInRangeXmlElementName : Not implemented."); };
    SyncResponse.prototype.ReadElementsFromJson = function (responseObject, service) { throw new Error("SyncResponse.ts - ReadElementsFromJson : Not implemented."); };
    SyncResponse.prototype.ReadElementsFromXmlJsObject = function (reader) { throw new Error("SyncResponse.ts - ReadElementsFromXmlJsObject : Not implemented."); };
    return SyncResponse;
}(ServiceResponse_1.ServiceResponse));
exports.SyncResponse = SyncResponse;
