// ews-javascript-api/js/ComplexProperties/Recurrence/Patterns/Recurrence_RelativeMonthlyPattern.js
var __extends = this.__extends || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    __.prototype = b.prototype;
    d.prototype = new __();
};
var Recurrence_IntervalPattern_1 = require("./Recurrence_IntervalPattern");
var RelativeMonthlyPattern = (function (_super) {
    __extends(RelativeMonthlyPattern, _super);
    function RelativeMonthlyPattern() {
        _super.apply(this, arguments);
    }
    RelativeMonthlyPattern.prototype.InternalValidate = function () { throw new Error("Recurrence_RelativeMonthlyPattern.ts - InternalValidate : Not implemented."); };
    RelativeMonthlyPattern.prototype.InternalWritePropertiesToXml = function (writer) { throw new Error("Recurrence_RelativeMonthlyPattern.ts - InternalWritePropertiesToXml : Not implemented."); };
    RelativeMonthlyPattern.prototype.LoadFromJson = function (jsonProperty, service) { throw new Error("Recurrence_RelativeMonthlyPattern.ts - LoadFromJson : Not implemented."); };
    RelativeMonthlyPattern.prototype.PatternToJson = function (service) { throw new Error("Recurrence_RelativeMonthlyPattern.ts - PatternToJson : Not implemented."); };
    RelativeMonthlyPattern.prototype.ReadElementsFromXmlJsObject = function (reader) { throw new Error("Recurrence_RelativeMonthlyPattern.ts - TryReadElementFromXmlJsObject : Not implemented."); };
    return RelativeMonthlyPattern;
})(Recurrence_IntervalPattern_1.IntervalPattern);
exports.RelativeMonthlyPattern = RelativeMonthlyPattern;
