// ews-javascript-api/js/ComplexProperties/Recurrence/Patterns/Recurrence_YearlyPattern.js
var __extends = this.__extends || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    __.prototype = b.prototype;
    d.prototype = new __();
};
var Recurrence_1 = require("./Recurrence");
var YearlyPattern = (function (_super) {
    __extends(YearlyPattern, _super);
    function YearlyPattern() {
        _super.apply(this, arguments);
    }
    YearlyPattern.prototype.InternalValidate = function () { throw new Error("Recurrence_YearlyPattern.ts - InternalValidate : Not implemented."); };
    YearlyPattern.prototype.InternalWritePropertiesToXml = function (writer) { throw new Error("Recurrence_YearlyPattern.ts - InternalWritePropertiesToXml : Not implemented."); };
    YearlyPattern.prototype.LoadFromJson = function (jsonProperty, service) { throw new Error("Recurrence_YearlyPattern.ts - LoadFromJson : Not implemented."); };
    YearlyPattern.prototype.PatternToJson = function (service) { throw new Error("Recurrence_YearlyPattern.ts - PatternToJson : Not implemented."); };
    YearlyPattern.prototype.ReadElementsFromXmlJsObject = function (reader) { throw new Error("Recurrence_YearlyPattern.ts - TryReadElementFromXmlJsObject : Not implemented."); };
    return YearlyPattern;
})(Recurrence_1.Recurrence);
exports.YearlyPattern = YearlyPattern;
//------------modulename->Microsoft.Exchange.WebServices.Data.Recurrence------------
