// ews-javascript-api/js/ComplexProperties/Recurrence/Patterns/Recurrence_WeeklyPattern.js
var __extends = this.__extends || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    __.prototype = b.prototype;
    d.prototype = new __();
};
var Recurrence_IntervalPattern_1 = require("./Recurrence_IntervalPattern");
var WeeklyPattern = (function (_super) {
    __extends(WeeklyPattern, _super);
    function WeeklyPattern() {
        _super.apply(this, arguments);
    }
    WeeklyPattern.prototype.DaysOfTheWeekChanged = function (complexProperty) { throw new Error("Recurrence_WeeklyPattern.ts - DaysOfTheWeekChanged : Not implemented."); };
    WeeklyPattern.prototype.InternalValidate = function () { throw new Error("Recurrence_WeeklyPattern.ts - InternalValidate : Not implemented."); };
    WeeklyPattern.prototype.InternalWritePropertiesToXml = function (writer) { throw new Error("Recurrence_WeeklyPattern.ts - InternalWritePropertiesToXml : Not implemented."); };
    WeeklyPattern.prototype.LoadFromJson = function (jsonProperty, service) { throw new Error("Recurrence_WeeklyPattern.ts - LoadFromJson : Not implemented."); };
    WeeklyPattern.prototype.PatternToJson = function (service) { throw new Error("Recurrence_WeeklyPattern.ts - PatternToJson : Not implemented."); };
    WeeklyPattern.prototype.ReadElementsFromXmlJsObject = function (reader) { throw new Error("Recurrence_WeeklyPattern.ts - TryReadElementFromXmlJsObject : Not implemented."); };
    return WeeklyPattern;
})(Recurrence_IntervalPattern_1.IntervalPattern);
exports.WeeklyPattern = WeeklyPattern;
//------------modulename->Microsoft.Exchange.WebServices.Data.Recurrence------------
