// ews-javascript-api/js/ComplexProperties/Recurrence/Patterns/Recurrence_RelativeYearlyPattern.js
var __extends = this.__extends || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    __.prototype = b.prototype;
    d.prototype = new __();
};
var Recurrence_1 = require("./Recurrence");
var RelativeYearlyPattern = (function (_super) {
    __extends(RelativeYearlyPattern, _super);
    function RelativeYearlyPattern() {
        _super.apply(this, arguments);
    }
    RelativeYearlyPattern.prototype.InternalValidate = function () { throw new Error("Recurrence_RelativeYearlyPattern.ts - InternalValidate : Not implemented."); };
    RelativeYearlyPattern.prototype.InternalWritePropertiesToXml = function (writer) { throw new Error("Recurrence_RelativeYearlyPattern.ts - InternalWritePropertiesToXml : Not implemented."); };
    RelativeYearlyPattern.prototype.LoadFromJson = function (jsonProperty, service) { throw new Error("Recurrence_RelativeYearlyPattern.ts - LoadFromJson : Not implemented."); };
    RelativeYearlyPattern.prototype.PatternToJson = function (service) { throw new Error("Recurrence_RelativeYearlyPattern.ts - PatternToJson : Not implemented."); };
    RelativeYearlyPattern.prototype.ReadElementsFromXmlJsObject = function (reader) { throw new Error("Recurrence_RelativeYearlyPattern.ts - TryReadElementFromXmlJsObject : Not implemented."); };
    return RelativeYearlyPattern;
})(Recurrence_1.Recurrence);
exports.RelativeYearlyPattern = RelativeYearlyPattern;
//------------modulename->Microsoft.Exchange.WebServices.Data.Recurrence------------
