// ews-javascript-api/js/ComplexProperties/Recurrence/Patterns/Recurrence_MonthlyPattern.js
var __extends = this.__extends || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    __.prototype = b.prototype;
    d.prototype = new __();
};
var Recurrence_IntervalPattern_1 = require("./Recurrence_IntervalPattern");
var MonthlyPattern = (function (_super) {
    __extends(MonthlyPattern, _super);
    function MonthlyPattern() {
        _super.apply(this, arguments);
    }
    MonthlyPattern.prototype.InternalValidate = function () { throw new Error("Recurrence_MonthlyPattern.ts - InternalValidate : Not implemented."); };
    MonthlyPattern.prototype.InternalWritePropertiesToXml = function (writer) { throw new Error("Recurrence_MonthlyPattern.ts - InternalWritePropertiesToXml : Not implemented."); };
    MonthlyPattern.prototype.LoadFromJson = function (jsonProperty, service) { throw new Error("Recurrence_MonthlyPattern.ts - LoadFromJson : Not implemented."); };
    MonthlyPattern.prototype.PatternToJson = function (service) { throw new Error("Recurrence_MonthlyPattern.ts - PatternToJson : Not implemented."); };
    MonthlyPattern.prototype.ReadElementsFromXmlJsObject = function (reader) { throw new Error("Recurrence_MonthlyPattern.ts - TryReadElementFromXmlJsObject : Not implemented."); };
    return MonthlyPattern;
})(Recurrence_IntervalPattern_1.IntervalPattern);
exports.MonthlyPattern = MonthlyPattern;
