// ews-javascript-api/js/Core/Requests/AddDelegateRequest.js
"use strict";
var __extends = (this && this.__extends) || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
};
var DelegateManagementRequestBase_1 = require("./DelegateManagementRequestBase");
/**
 * ## *Not Implemented*
 */
var AddDelegateRequest = (function (_super) {
    __extends(AddDelegateRequest, _super);
    function AddDelegateRequest() {
        _super.apply(this, arguments);
    }
    AddDelegateRequest.prototype.CreateResponse = function () { throw new Error("AddDelegateRequest.ts - CreateResponse : Not implemented."); };
    AddDelegateRequest.prototype.GetMinimumRequiredServerVersion = function () { throw new Error("AddDelegateRequest.ts - GetMinimumRequiredServerVersion : Not implemented."); };
    AddDelegateRequest.prototype.GetResponseXmlElementName = function () { throw new Error("AddDelegateRequest.ts - GetResponseXmlElementName : Not implemented."); };
    AddDelegateRequest.prototype.GetXmlElementName = function () { throw new Error("AddDelegateRequest.ts - GetXmlElementName : Not implemented."); };
    AddDelegateRequest.prototype.Validate = function () { throw new Error("AddDelegateRequest.ts - Validate : Not implemented."); };
    AddDelegateRequest.prototype.WriteElementsToXml = function (writer) { throw new Error("AddDelegateRequest.ts - WriteElementsToXml : Not implemented."); };
    return AddDelegateRequest;
}(DelegateManagementRequestBase_1.DelegateManagementRequestBase));
exports.AddDelegateRequest = AddDelegateRequest;
