// ews-javascript-api/js/Core/Requests/ConvertIdRequest.js
"use strict";
var __extends = (this && this.__extends) || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
};
var MultiResponseServiceRequest_1 = require("./MultiResponseServiceRequest");
/**
 * ## *Not Implemented*
 */
var ConvertIdRequest = (function (_super) {
    __extends(ConvertIdRequest, _super);
    function ConvertIdRequest() {
        _super.apply(this, arguments);
    }
    ConvertIdRequest.prototype.CreateServiceResponse = function (service, responseIndex) { throw new Error("ConvertIdRequest.ts - CreateServiceResponse : Not implemented."); };
    ConvertIdRequest.prototype.GetExpectedResponseMessageCount = function () { throw new Error("ConvertIdRequest.ts - GetExpectedResponseMessageCount : Not implemented."); };
    ConvertIdRequest.prototype.GetMinimumRequiredServerVersion = function () { throw new Error("ConvertIdRequest.ts - GetMinimumRequiredServerVersion : Not implemented."); };
    ConvertIdRequest.prototype.GetResponseMessageXmlElementName = function () { throw new Error("ConvertIdRequest.ts - GetResponseMessageXmlElementName : Not implemented."); };
    ConvertIdRequest.prototype.GetResponseXmlElementName = function () { throw new Error("ConvertIdRequest.ts - GetResponseXmlElementName : Not implemented."); };
    ConvertIdRequest.prototype.GetXmlElementName = function () { throw new Error("ConvertIdRequest.ts - GetXmlElementName : Not implemented."); };
    ConvertIdRequest.prototype.Validate = function () { throw new Error("ConvertIdRequest.ts - Validate : Not implemented."); };
    ConvertIdRequest.prototype.WriteElementsToXml = function (writer) { throw new Error("ConvertIdRequest.ts - WriteElementsToXml : Not implemented."); };
    return ConvertIdRequest;
}(MultiResponseServiceRequest_1.MultiResponseServiceRequest));
exports.ConvertIdRequest = ConvertIdRequest;
