// ews-javascript-api/js/Core/Requests/DelegateManagementRequestBase.js
"use strict";
var __extends = (this && this.__extends) || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
};
var SimpleServiceRequestBase_1 = require("./SimpleServiceRequestBase");
/**
 * ## *Not Implemented*
 */
var DelegateManagementRequestBase = (function (_super) {
    __extends(DelegateManagementRequestBase, _super);
    function DelegateManagementRequestBase() {
        _super.apply(this, arguments);
    }
    DelegateManagementRequestBase.prototype.CreateResponse = function () { throw new Error("DelegateManagementRequestBase.ts - CreateResponse : Not implemented."); };
    DelegateManagementRequestBase.prototype.Execute = function () { throw new Error("DelegateManagementRequestBase.ts - Execute : Not implemented."); };
    DelegateManagementRequestBase.prototype.ParseResponse = function (reader) { throw new Error("DelegateManagementRequestBase.ts - ParseResponse : Not implemented."); };
    DelegateManagementRequestBase.prototype.Validate = function () { throw new Error("DelegateManagementRequestBase.ts - Validate : Not implemented."); };
    DelegateManagementRequestBase.prototype.WriteElementsToXml = function (writer) { throw new Error("DelegateManagementRequestBase.ts - WriteElementsToXml : Not implemented."); };
    return DelegateManagementRequestBase;
}(SimpleServiceRequestBase_1.SimpleServiceRequestBase));
exports.DelegateManagementRequestBase = DelegateManagementRequestBase;
