// ews-javascript-api/js/Core/Requests/GetClientAccessTokenRequest.js
"use strict";
var __extends = (this && this.__extends) || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
};
var MultiResponseServiceRequest_1 = require("./MultiResponseServiceRequest");
/**
 * ## *Not Implemented*
 */
var GetClientAccessTokenRequest = (function (_super) {
    __extends(GetClientAccessTokenRequest, _super);
    function GetClientAccessTokenRequest() {
        _super.apply(this, arguments);
    }
    GetClientAccessTokenRequest.prototype.CreateServiceResponse = function (service, responseIndex) { throw new Error("GetClientAccessTokenRequest.ts - CreateServiceResponse : Not implemented."); };
    GetClientAccessTokenRequest.prototype.GetExpectedResponseMessageCount = function () { throw new Error("GetClientAccessTokenRequest.ts - GetExpectedResponseMessageCount : Not implemented."); };
    GetClientAccessTokenRequest.prototype.GetMinimumRequiredServerVersion = function () { throw new Error("GetClientAccessTokenRequest.ts - GetMinimumRequiredServerVersion : Not implemented."); };
    GetClientAccessTokenRequest.prototype.GetResponseMessageXmlElementName = function () { throw new Error("GetClientAccessTokenRequest.ts - GetResponseMessageXmlElementName : Not implemented."); };
    GetClientAccessTokenRequest.prototype.GetResponseXmlElementName = function () { throw new Error("GetClientAccessTokenRequest.ts - GetResponseXmlElementName : Not implemented."); };
    GetClientAccessTokenRequest.prototype.GetXmlElementName = function () { throw new Error("GetClientAccessTokenRequest.ts - GetXmlElementName : Not implemented."); };
    GetClientAccessTokenRequest.prototype.Validate = function () { throw new Error("GetClientAccessTokenRequest.ts - Validate : Not implemented."); };
    GetClientAccessTokenRequest.prototype.WriteElementsToXml = function (writer) { throw new Error("GetClientAccessTokenRequest.ts - WriteElementsToXml : Not implemented."); };
    return GetClientAccessTokenRequest;
}(MultiResponseServiceRequest_1.MultiResponseServiceRequest));
exports.GetClientAccessTokenRequest = GetClientAccessTokenRequest;
