// ews-javascript-api/js/Core/Requests/GetDelegateRequest.js
"use strict";
var __extends = (this && this.__extends) || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
};
var DelegateManagementRequestBase_1 = require("./DelegateManagementRequestBase");
/**
 * ## *Not Implemented*
 */
var GetDelegateRequest = (function (_super) {
    __extends(GetDelegateRequest, _super);
    function GetDelegateRequest() {
        _super.apply(this, arguments);
    }
    GetDelegateRequest.prototype.CreateResponse = function () { throw new Error("GetDelegateRequest.ts - CreateResponse : Not implemented."); };
    GetDelegateRequest.prototype.GetMinimumRequiredServerVersion = function () { throw new Error("GetDelegateRequest.ts - GetMinimumRequiredServerVersion : Not implemented."); };
    GetDelegateRequest.prototype.GetResponseXmlElementName = function () { throw new Error("GetDelegateRequest.ts - GetResponseXmlElementName : Not implemented."); };
    GetDelegateRequest.prototype.GetXmlElementName = function () { throw new Error("GetDelegateRequest.ts - GetXmlElementName : Not implemented."); };
    GetDelegateRequest.prototype.WriteAttributesToXml = function (writer) { throw new Error("GetDelegateRequest.ts - WriteAttributesToXml : Not implemented."); };
    GetDelegateRequest.prototype.WriteElementsToXml = function (writer) { throw new Error("GetDelegateRequest.ts - WriteElementsToXml : Not implemented."); };
    return GetDelegateRequest;
}(DelegateManagementRequestBase_1.DelegateManagementRequestBase));
exports.GetDelegateRequest = GetDelegateRequest;
