// ews-javascript-api/js/Core/Requests/GetEventsRequest.js
"use strict";
var __extends = (this && this.__extends) || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
};
var MultiResponseServiceRequest_1 = require("./MultiResponseServiceRequest");
/**
 * ## *Not Implemented*
 */
var GetEventsRequest = (function (_super) {
    __extends(GetEventsRequest, _super);
    function GetEventsRequest() {
        _super.apply(this, arguments);
    }
    GetEventsRequest.prototype.CreateServiceResponse = function (service, responseIndex) { throw new Error("GetEventsRequest.ts - CreateServiceResponse : Not implemented."); };
    GetEventsRequest.prototype.GetExpectedResponseMessageCount = function () { throw new Error("GetEventsRequest.ts - GetExpectedResponseMessageCount : Not implemented."); };
    GetEventsRequest.prototype.GetMinimumRequiredServerVersion = function () { throw new Error("GetEventsRequest.ts - GetMinimumRequiredServerVersion : Not implemented."); };
    GetEventsRequest.prototype.GetResponseMessageXmlElementName = function () { throw new Error("GetEventsRequest.ts - GetResponseMessageXmlElementName : Not implemented."); };
    GetEventsRequest.prototype.GetResponseXmlElementName = function () { throw new Error("GetEventsRequest.ts - GetResponseXmlElementName : Not implemented."); };
    GetEventsRequest.prototype.GetXmlElementName = function () { throw new Error("GetEventsRequest.ts - GetXmlElementName : Not implemented."); };
    GetEventsRequest.prototype.Validate = function () { throw new Error("GetEventsRequest.ts - Validate : Not implemented."); };
    GetEventsRequest.prototype.WriteElementsToXml = function (writer) { throw new Error("GetEventsRequest.ts - WriteElementsToXml : Not implemented."); };
    return GetEventsRequest;
}(MultiResponseServiceRequest_1.MultiResponseServiceRequest));
exports.GetEventsRequest = GetEventsRequest;
