// ews-javascript-api/js/Core/Requests/GetInboxRulesRequest.js
"use strict";
var __extends = (this && this.__extends) || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
};
var SimpleServiceRequestBase_1 = require("./SimpleServiceRequestBase");
/**
 * ## *Not Implemented*
 */
var GetInboxRulesRequest = (function (_super) {
    __extends(GetInboxRulesRequest, _super);
    function GetInboxRulesRequest() {
        _super.apply(this, arguments);
    }
    GetInboxRulesRequest.prototype.Execute = function () { throw new Error("GetInboxRulesRequest.ts - Execute : Not implemented."); };
    GetInboxRulesRequest.prototype.GetMinimumRequiredServerVersion = function () { throw new Error("GetInboxRulesRequest.ts - GetMinimumRequiredServerVersion : Not implemented."); };
    GetInboxRulesRequest.prototype.GetResponseXmlElementName = function () { throw new Error("GetInboxRulesRequest.ts - GetResponseXmlElementName : Not implemented."); };
    GetInboxRulesRequest.prototype.GetXmlElementName = function () { throw new Error("GetInboxRulesRequest.ts - GetXmlElementName : Not implemented."); };
    GetInboxRulesRequest.prototype.ParseResponse = function (reader) { throw new Error("GetInboxRulesRequest.ts - ParseResponse : Not implemented."); };
    GetInboxRulesRequest.prototype.WriteElementsToXml = function (writer) { throw new Error("GetInboxRulesRequest.ts - WriteElementsToXml : Not implemented."); };
    return GetInboxRulesRequest;
}(SimpleServiceRequestBase_1.SimpleServiceRequestBase));
exports.GetInboxRulesRequest = GetInboxRulesRequest;
