// ews-javascript-api/js/Core/Requests/GetRoomsRequest.js
"use strict";
var __extends = (this && this.__extends) || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
};
var SimpleServiceRequestBase_1 = require("./SimpleServiceRequestBase");
/**
 * ## *Not Implemented*
 */
var GetRoomsRequest = (function (_super) {
    __extends(GetRoomsRequest, _super);
    function GetRoomsRequest() {
        _super.apply(this, arguments);
    }
    GetRoomsRequest.prototype.Execute = function () { throw new Error("GetRoomsRequest.ts - Execute : Not implemented."); };
    GetRoomsRequest.prototype.GetMinimumRequiredServerVersion = function () { throw new Error("GetRoomsRequest.ts - GetMinimumRequiredServerVersion : Not implemented."); };
    GetRoomsRequest.prototype.GetResponseXmlElementName = function () { throw new Error("GetRoomsRequest.ts - GetResponseXmlElementName : Not implemented."); };
    GetRoomsRequest.prototype.GetXmlElementName = function () { throw new Error("GetRoomsRequest.ts - GetXmlElementName : Not implemented."); };
    GetRoomsRequest.prototype.ParseResponse = function (reader) { throw new Error("GetRoomsRequest.ts - ParseResponse : Not implemented."); };
    GetRoomsRequest.prototype.WriteElementsToXml = function (writer) { throw new Error("GetRoomsRequest.ts - WriteElementsToXml : Not implemented."); };
    return GetRoomsRequest;
}(SimpleServiceRequestBase_1.SimpleServiceRequestBase));
exports.GetRoomsRequest = GetRoomsRequest;
