// ews-javascript-api/js/Core/Requests/GetRoomListsRequest.js
"use strict";
var __extends = (this && this.__extends) || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
};
var SimpleServiceRequestBase_1 = require("./SimpleServiceRequestBase");
var GetRoomListsRequest = (function (_super) {
    __extends(GetRoomListsRequest, _super);
    function GetRoomListsRequest() {
        _super.apply(this, arguments);
    }
    GetRoomListsRequest.prototype.Execute = function () { throw new Error("GetRoomListsRequest.ts - Execute : Not implemented."); };
    GetRoomListsRequest.prototype.GetMinimumRequiredServerVersion = function () { throw new Error("GetRoomListsRequest.ts - GetMinimumRequiredServerVersion : Not implemented."); };
    GetRoomListsRequest.prototype.GetResponseXmlElementName = function () { throw new Error("GetRoomListsRequest.ts - GetResponseXmlElementName : Not implemented."); };
    GetRoomListsRequest.prototype.GetXmlElementName = function () { throw new Error("GetRoomListsRequest.ts - GetXmlElementName : Not implemented."); };
    GetRoomListsRequest.prototype.ParseResponse = function (reader) { throw new Error("GetRoomListsRequest.ts - ParseResponse : Not implemented."); };
    GetRoomListsRequest.prototype.WriteElementsToXml = function (writer) { throw new Error("GetRoomListsRequest.ts - WriteElementsToXml : Not implemented."); };
    return GetRoomListsRequest;
}(SimpleServiceRequestBase_1.SimpleServiceRequestBase));
exports.GetRoomListsRequest = GetRoomListsRequest;
