// ews-javascript-api/js/Core/Requests/GetStreamingEventsRequest.js
"use strict";
var __extends = (this && this.__extends) || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
};
var HangingServiceRequestBase_1 = require("./HangingServiceRequestBase");
/**
 * ## *Not Implemented*
 */
var GetStreamingEventsRequest = (function (_super) {
    __extends(GetStreamingEventsRequest, _super);
    function GetStreamingEventsRequest() {
        _super.apply(this, arguments);
    }
    GetStreamingEventsRequest.prototype.GetMinimumRequiredServerVersion = function () { throw new Error("GetStreamingEventsRequest.ts - GetMinimumRequiredServerVersion : Not implemented."); };
    GetStreamingEventsRequest.prototype.GetResponseXmlElementName = function () { throw new Error("GetStreamingEventsRequest.ts - GetResponseXmlElementName : Not implemented."); };
    GetStreamingEventsRequest.prototype.GetXmlElementName = function () { throw new Error("GetStreamingEventsRequest.ts - GetXmlElementName : Not implemented."); };
    GetStreamingEventsRequest.prototype.ParseResponse = function (reader) { throw new Error("GetStreamingEventsRequest.ts - ParseResponse : Not implemented."); };
    GetStreamingEventsRequest.prototype.WriteElementsToXml = function (writer) { throw new Error("GetStreamingEventsRequest.ts - WriteElementsToXml : Not implemented."); };
    GetStreamingEventsRequest.HeartbeatFrequencyDefault = 45000;
    return GetStreamingEventsRequest;
}(HangingServiceRequestBase_1.HangingServiceRequestBase));
exports.GetStreamingEventsRequest = GetStreamingEventsRequest;
