// ews-javascript-api/js/Core/Requests/HangingServiceRequestBase.js
"use strict";
var __extends = (this && this.__extends) || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
};
var ServiceRequestBase_1 = require("./ServiceRequestBase");
/**
 * ## *Not Implemented*
 */
var HangingServiceRequestBase = (function (_super) {
    __extends(HangingServiceRequestBase, _super);
    function HangingServiceRequestBase() {
        _super.apply(this, arguments);
        /**
         * @internal Occurs when the hanging request is disconnected.
         */
        this.OnDisconnect = [];
    }
    HangingServiceRequestBase.prototype.Disconnect = function (reason, exception) { throw new Error("HangingServiceRequestBase.ts - Disconnect : Not implemented."); };
    HangingServiceRequestBase.prototype.InternalExecute = function () { throw new Error("HangingServiceRequestBase.ts - InternalExecute : Not implemented."); };
    HangingServiceRequestBase.prototype.InternalOnConnect = function () { throw new Error("HangingServiceRequestBase.ts - InternalOnConnect : Not implemented."); };
    HangingServiceRequestBase.prototype.InternalOnDisconnect = function (reason, exception) { throw new Error("HangingServiceRequestBase.ts - InternalOnDisconnect : Not implemented."); };
    HangingServiceRequestBase.prototype.ParseResponses = function (state) { throw new Error("HangingServiceRequestBase.ts - ParseResponses : Not implemented."); };
    HangingServiceRequestBase.prototype.ReadPreamble = function (ewsXmlReader) { throw new Error("HangingServiceRequestBase.ts - ReadPreamble : Not implemented."); };
    HangingServiceRequestBase.BufferSize = 4096;
    return HangingServiceRequestBase;
}(ServiceRequestBase_1.ServiceRequestBase));
exports.HangingServiceRequestBase = HangingServiceRequestBase;
