// ews-javascript-api/js/Core/Requests/RemoveDelegateRequest.js
"use strict";
var __extends = (this && this.__extends) || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
};
var DelegateManagementRequestBase_1 = require("./DelegateManagementRequestBase");
/**
 * ## *Not Implemented*
 */
var RemoveDelegateRequest = (function (_super) {
    __extends(RemoveDelegateRequest, _super);
    function RemoveDelegateRequest() {
        _super.apply(this, arguments);
    }
    RemoveDelegateRequest.prototype.CreateResponse = function () { throw new Error("RemoveDelegateRequest.ts - CreateResponse : Not implemented."); };
    RemoveDelegateRequest.prototype.GetMinimumRequiredServerVersion = function () { throw new Error("RemoveDelegateRequest.ts - GetMinimumRequiredServerVersion : Not implemented."); };
    RemoveDelegateRequest.prototype.GetResponseXmlElementName = function () { throw new Error("RemoveDelegateRequest.ts - GetResponseXmlElementName : Not implemented."); };
    RemoveDelegateRequest.prototype.GetXmlElementName = function () { throw new Error("RemoveDelegateRequest.ts - GetXmlElementName : Not implemented."); };
    RemoveDelegateRequest.prototype.Validate = function () { throw new Error("RemoveDelegateRequest.ts - Validate : Not implemented."); };
    RemoveDelegateRequest.prototype.WriteElementsToXml = function (writer) { throw new Error("RemoveDelegateRequest.ts - WriteElementsToXml : Not implemented."); };
    return RemoveDelegateRequest;
}(DelegateManagementRequestBase_1.DelegateManagementRequestBase));
exports.RemoveDelegateRequest = RemoveDelegateRequest;
