// ews-javascript-api/js/Core/Requests/SetTeamMailboxRequest.js
"use strict";
var __extends = (this && this.__extends) || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
};
var SimpleServiceRequestBase_1 = require("./SimpleServiceRequestBase");
/**
 * ## *Not Implemented*
 */
var SetTeamMailboxRequest = (function (_super) {
    __extends(SetTeamMailboxRequest, _super);
    function SetTeamMailboxRequest() {
        _super.apply(this, arguments);
    }
    SetTeamMailboxRequest.prototype.Execute = function () { throw new Error("SetTeamMailboxRequest.ts - Execute : Not implemented."); };
    SetTeamMailboxRequest.prototype.GetMinimumRequiredServerVersion = function () { throw new Error("SetTeamMailboxRequest.ts - GetMinimumRequiredServerVersion : Not implemented."); };
    SetTeamMailboxRequest.prototype.GetResponseXmlElementName = function () { throw new Error("SetTeamMailboxRequest.ts - GetResponseXmlElementName : Not implemented."); };
    SetTeamMailboxRequest.prototype.GetXmlElementName = function () { throw new Error("SetTeamMailboxRequest.ts - GetXmlElementName : Not implemented."); };
    SetTeamMailboxRequest.prototype.ParseResponse = function (reader) { throw new Error("SetTeamMailboxRequest.ts - ParseResponse : Not implemented."); };
    SetTeamMailboxRequest.prototype.WriteElementsToXml = function (writer) { throw new Error("SetTeamMailboxRequest.ts - WriteElementsToXml : Not implemented."); };
    return SetTeamMailboxRequest;
}(SimpleServiceRequestBase_1.SimpleServiceRequestBase));
exports.SetTeamMailboxRequest = SetTeamMailboxRequest;
