// ews-javascript-api/js/Core/Requests/SubscribeToStreamingNotificationsRequest.js
"use strict";
var __extends = (this && this.__extends) || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
};
var SubscribeRequest_1 = require("./SubscribeRequest");
/**
 * ## *Not Implemented*
 */
var SubscribeToStreamingNotificationsRequest = (function (_super) {
    __extends(SubscribeToStreamingNotificationsRequest, _super);
    function SubscribeToStreamingNotificationsRequest() {
        _super.apply(this, arguments);
    }
    SubscribeToStreamingNotificationsRequest.prototype.AddJsonProperties = function (jsonSubscribeRequest, service) { throw new Error("SubscribeToStreamingNotificationsRequest.ts - AddJsonProperties : Not implemented."); };
    SubscribeToStreamingNotificationsRequest.prototype.CreateServiceResponse = function (service, responseIndex) { throw new Error("SubscribeToStreamingNotificationsRequest.ts - CreateServiceResponse : Not implemented."); };
    SubscribeToStreamingNotificationsRequest.prototype.GetMinimumRequiredServerVersion = function () { throw new Error("SubscribeToStreamingNotificationsRequest.ts - GetMinimumRequiredServerVersion : Not implemented."); };
    SubscribeToStreamingNotificationsRequest.prototype.GetSubscriptionXmlElementName = function () { throw new Error("SubscribeToStreamingNotificationsRequest.ts - GetSubscriptionXmlElementName : Not implemented."); };
    SubscribeToStreamingNotificationsRequest.prototype.InternalWriteElementsToXml = function (writer) { throw new Error("SubscribeToStreamingNotificationsRequest.ts - InternalWriteElementsToXml : Not implemented."); };
    SubscribeToStreamingNotificationsRequest.prototype.Validate = function () { throw new Error("SubscribeToStreamingNotificationsRequest.ts - Validate : Not implemented."); };
    return SubscribeToStreamingNotificationsRequest;
}(SubscribeRequest_1.SubscribeRequest));
exports.SubscribeToStreamingNotificationsRequest = SubscribeToStreamingNotificationsRequest;
