// ews-javascript-api/js/Core/Requests/SubscribeToPushNotificationsRequest.js
"use strict";
var __extends = (this && this.__extends) || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
};
var SubscribeRequest_1 = require("./SubscribeRequest");
/**
 * ## *Not Implemented*
 */
var SubscribeToPushNotificationsRequest = (function (_super) {
    __extends(SubscribeToPushNotificationsRequest, _super);
    function SubscribeToPushNotificationsRequest() {
        _super.apply(this, arguments);
    }
    SubscribeToPushNotificationsRequest.prototype.AddJsonProperties = function (jsonSubscribeRequest, service) { throw new Error("SubscribeToPushNotificationsRequest.ts - AddJsonProperties : Not implemented."); };
    SubscribeToPushNotificationsRequest.prototype.CreateServiceResponse = function (service, responseIndex) { throw new Error("SubscribeToPushNotificationsRequest.ts - CreateServiceResponse : Not implemented."); };
    SubscribeToPushNotificationsRequest.prototype.GetMinimumRequiredServerVersion = function () { throw new Error("SubscribeToPushNotificationsRequest.ts - GetMinimumRequiredServerVersion : Not implemented."); };
    SubscribeToPushNotificationsRequest.prototype.GetSubscriptionXmlElementName = function () { throw new Error("SubscribeToPushNotificationsRequest.ts - GetSubscriptionXmlElementName : Not implemented."); };
    SubscribeToPushNotificationsRequest.prototype.InternalWriteElementsToXml = function (writer) { throw new Error("SubscribeToPushNotificationsRequest.ts - InternalWriteElementsToXml : Not implemented."); };
    SubscribeToPushNotificationsRequest.prototype.Validate = function () { throw new Error("SubscribeToPushNotificationsRequest.ts - Validate : Not implemented."); };
    return SubscribeToPushNotificationsRequest;
}(SubscribeRequest_1.SubscribeRequest));
exports.SubscribeToPushNotificationsRequest = SubscribeToPushNotificationsRequest;
