// ews-javascript-api/js/Core/Requests/SubscribeToPullNotificationsRequest.js
"use strict";
var __extends = (this && this.__extends) || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
};
var SubscribeRequest_1 = require("./SubscribeRequest");
/**
 * ## *Not Implemented*
 */
var SubscribeToPullNotificationsRequest = (function (_super) {
    __extends(SubscribeToPullNotificationsRequest, _super);
    function SubscribeToPullNotificationsRequest() {
        _super.apply(this, arguments);
    }
    SubscribeToPullNotificationsRequest.prototype.AddJsonProperties = function (jsonSubscribeRequest, service) { throw new Error("SubscribeToPullNotificationsRequest.ts - AddJsonProperties : Not implemented."); };
    SubscribeToPullNotificationsRequest.prototype.CreateServiceResponse = function (service, responseIndex) { throw new Error("SubscribeToPullNotificationsRequest.ts - CreateServiceResponse : Not implemented."); };
    SubscribeToPullNotificationsRequest.prototype.GetMinimumRequiredServerVersion = function () { throw new Error("SubscribeToPullNotificationsRequest.ts - GetMinimumRequiredServerVersion : Not implemented."); };
    SubscribeToPullNotificationsRequest.prototype.GetSubscriptionXmlElementName = function () { throw new Error("SubscribeToPullNotificationsRequest.ts - GetSubscriptionXmlElementName : Not implemented."); };
    SubscribeToPullNotificationsRequest.prototype.InternalWriteElementsToXml = function (writer) { throw new Error("SubscribeToPullNotificationsRequest.ts - InternalWriteElementsToXml : Not implemented."); };
    SubscribeToPullNotificationsRequest.prototype.Validate = function () { throw new Error("SubscribeToPullNotificationsRequest.ts - Validate : Not implemented."); };
    return SubscribeToPullNotificationsRequest;
}(SubscribeRequest_1.SubscribeRequest));
exports.SubscribeToPullNotificationsRequest = SubscribeToPullNotificationsRequest;
