// ews-javascript-api/js/Core/Requests/SubscribeRequest.js
"use strict";
var __extends = (this && this.__extends) || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
};
var MultiResponseServiceRequest_1 = require("./MultiResponseServiceRequest");
/**
 * ## *Not Implemented*
 */
var SubscribeRequest = (function (_super) {
    __extends(SubscribeRequest, _super);
    function SubscribeRequest() {
        _super.apply(this, arguments);
    }
    SubscribeRequest.prototype.AddJsonProperties = function (jsonSubscribeRequest, service) { throw new Error("SubscribeRequest.ts - AddJsonProperties : Not implemented."); };
    SubscribeRequest.prototype.GetExpectedResponseMessageCount = function () { throw new Error("SubscribeRequest.ts - GetExpectedResponseMessageCount : Not implemented."); };
    SubscribeRequest.prototype.GetResponseMessageXmlElementName = function () { throw new Error("SubscribeRequest.ts - GetResponseMessageXmlElementName : Not implemented."); };
    SubscribeRequest.prototype.GetResponseXmlElementName = function () { throw new Error("SubscribeRequest.ts - GetResponseXmlElementName : Not implemented."); };
    SubscribeRequest.prototype.GetSubscriptionXmlElementName = function () { throw new Error("SubscribeRequest.ts - GetSubscriptionXmlElementName : Not implemented."); };
    SubscribeRequest.prototype.GetXmlElementName = function () { throw new Error("SubscribeRequest.ts - GetXmlElementName : Not implemented."); };
    SubscribeRequest.prototype.InternalWriteElementsToXml = function (writer) { throw new Error("SubscribeRequest.ts - InternalWriteElementsToXml : Not implemented."); };
    SubscribeRequest.prototype.Validate = function () { throw new Error("SubscribeRequest.ts - Validate : Not implemented."); };
    SubscribeRequest.prototype.WriteElementsToXml = function (writer) { throw new Error("SubscribeRequest.ts - WriteElementsToXml : Not implemented."); };
    return SubscribeRequest;
}(MultiResponseServiceRequest_1.MultiResponseServiceRequest));
exports.SubscribeRequest = SubscribeRequest;
