// ews-javascript-api/js/Core/Requests/SyncFolderItemsRequest.js
"use strict";
var __extends = (this && this.__extends) || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
};
var MultiResponseServiceRequest_1 = require("./MultiResponseServiceRequest");
/**
 * ## *Not Implemented*
 */
var SyncFolderItemsRequest = (function (_super) {
    __extends(SyncFolderItemsRequest, _super);
    function SyncFolderItemsRequest() {
        _super.apply(this, arguments);
    }
    SyncFolderItemsRequest.prototype.CreateServiceResponse = function (service, responseIndex) { throw new Error("SyncFolderItemsRequest.ts - CreateServiceResponse : Not implemented."); };
    SyncFolderItemsRequest.prototype.GetExpectedResponseMessageCount = function () { throw new Error("SyncFolderItemsRequest.ts - GetExpectedResponseMessageCount : Not implemented."); };
    SyncFolderItemsRequest.prototype.GetMinimumRequiredServerVersion = function () { throw new Error("SyncFolderItemsRequest.ts - GetMinimumRequiredServerVersion : Not implemented."); };
    SyncFolderItemsRequest.prototype.GetResponseMessageXmlElementName = function () { throw new Error("SyncFolderItemsRequest.ts - GetResponseMessageXmlElementName : Not implemented."); };
    SyncFolderItemsRequest.prototype.GetResponseXmlElementName = function () { throw new Error("SyncFolderItemsRequest.ts - GetResponseXmlElementName : Not implemented."); };
    SyncFolderItemsRequest.prototype.GetXmlElementName = function () { throw new Error("SyncFolderItemsRequest.ts - GetXmlElementName : Not implemented."); };
    SyncFolderItemsRequest.prototype.Validate = function () { throw new Error("SyncFolderItemsRequest.ts - Validate : Not implemented."); };
    SyncFolderItemsRequest.prototype.WriteElementsToXml = function (writer) { throw new Error("SyncFolderItemsRequest.ts - WriteElementsToXml : Not implemented."); };
    return SyncFolderItemsRequest;
}(MultiResponseServiceRequest_1.MultiResponseServiceRequest));
exports.SyncFolderItemsRequest = SyncFolderItemsRequest;
