// ews-javascript-api/js/Core/Requests/SyncFolderHierarchyRequest.js
"use strict";
var __extends = (this && this.__extends) || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
};
var MultiResponseServiceRequest_1 = require("./MultiResponseServiceRequest");
/**
 * ## *Not Implemented*
 */
var SyncFolderHierarchyRequest = (function (_super) {
    __extends(SyncFolderHierarchyRequest, _super);
    function SyncFolderHierarchyRequest() {
        _super.apply(this, arguments);
    }
    SyncFolderHierarchyRequest.prototype.CreateServiceResponse = function (service, responseIndex) { throw new Error("SyncFolderHierarchyRequest.ts - CreateServiceResponse : Not implemented."); };
    SyncFolderHierarchyRequest.prototype.GetExpectedResponseMessageCount = function () { throw new Error("SyncFolderHierarchyRequest.ts - GetExpectedResponseMessageCount : Not implemented."); };
    SyncFolderHierarchyRequest.prototype.GetMinimumRequiredServerVersion = function () { throw new Error("SyncFolderHierarchyRequest.ts - GetMinimumRequiredServerVersion : Not implemented."); };
    SyncFolderHierarchyRequest.prototype.GetResponseMessageXmlElementName = function () { throw new Error("SyncFolderHierarchyRequest.ts - GetResponseMessageXmlElementName : Not implemented."); };
    SyncFolderHierarchyRequest.prototype.GetResponseXmlElementName = function () { throw new Error("SyncFolderHierarchyRequest.ts - GetResponseXmlElementName : Not implemented."); };
    SyncFolderHierarchyRequest.prototype.GetXmlElementName = function () { throw new Error("SyncFolderHierarchyRequest.ts - GetXmlElementName : Not implemented."); };
    SyncFolderHierarchyRequest.prototype.Validate = function () { throw new Error("SyncFolderHierarchyRequest.ts - Validate : Not implemented."); };
    SyncFolderHierarchyRequest.prototype.WriteElementsToXml = function (writer) { throw new Error("SyncFolderHierarchyRequest.ts - WriteElementsToXml : Not implemented."); };
    return SyncFolderHierarchyRequest;
}(MultiResponseServiceRequest_1.MultiResponseServiceRequest));
exports.SyncFolderHierarchyRequest = SyncFolderHierarchyRequest;
