// ews-javascript-api/js/Core/Requests/UnpinTeamMailboxRequest.js
"use strict";
var __extends = (this && this.__extends) || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
};
var SimpleServiceRequestBase_1 = require("./SimpleServiceRequestBase");
/**
 * ## *Not Implemented*
 */
var UnpinTeamMailboxRequest = (function (_super) {
    __extends(UnpinTeamMailboxRequest, _super);
    function UnpinTeamMailboxRequest() {
        _super.apply(this, arguments);
    }
    UnpinTeamMailboxRequest.prototype.Execute = function () { throw new Error("UnpinTeamMailboxRequest.ts - Execute : Not implemented."); };
    UnpinTeamMailboxRequest.prototype.GetMinimumRequiredServerVersion = function () { throw new Error("UnpinTeamMailboxRequest.ts - GetMinimumRequiredServerVersion : Not implemented."); };
    UnpinTeamMailboxRequest.prototype.GetResponseXmlElementName = function () { throw new Error("UnpinTeamMailboxRequest.ts - GetResponseXmlElementName : Not implemented."); };
    UnpinTeamMailboxRequest.prototype.GetXmlElementName = function () { throw new Error("UnpinTeamMailboxRequest.ts - GetXmlElementName : Not implemented."); };
    UnpinTeamMailboxRequest.prototype.ParseResponse = function (reader) { throw new Error("UnpinTeamMailboxRequest.ts - ParseResponse : Not implemented."); };
    UnpinTeamMailboxRequest.prototype.WriteElementsToXml = function (writer) { throw new Error("UnpinTeamMailboxRequest.ts - WriteElementsToXml : Not implemented."); };
    return UnpinTeamMailboxRequest;
}(SimpleServiceRequestBase_1.SimpleServiceRequestBase));
exports.UnpinTeamMailboxRequest = UnpinTeamMailboxRequest;
