// ews-javascript-api/js/Core/Requests/UnsubscribeRequest.js
"use strict";
var __extends = (this && this.__extends) || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
};
var MultiResponseServiceRequest_1 = require("./MultiResponseServiceRequest");
/**
 * ## *Not Implemented*
 */
var UnsubscribeRequest = (function (_super) {
    __extends(UnsubscribeRequest, _super);
    function UnsubscribeRequest() {
        _super.apply(this, arguments);
    }
    UnsubscribeRequest.prototype.CreateServiceResponse = function (service, responseIndex) { throw new Error("UnsubscribeRequest.ts - CreateServiceResponse : Not implemented."); };
    UnsubscribeRequest.prototype.GetExpectedResponseMessageCount = function () { throw new Error("UnsubscribeRequest.ts - GetExpectedResponseMessageCount : Not implemented."); };
    UnsubscribeRequest.prototype.GetMinimumRequiredServerVersion = function () { throw new Error("UnsubscribeRequest.ts - GetMinimumRequiredServerVersion : Not implemented."); };
    UnsubscribeRequest.prototype.GetResponseMessageXmlElementName = function () { throw new Error("UnsubscribeRequest.ts - GetResponseMessageXmlElementName : Not implemented."); };
    UnsubscribeRequest.prototype.GetResponseXmlElementName = function () { throw new Error("UnsubscribeRequest.ts - GetResponseXmlElementName : Not implemented."); };
    UnsubscribeRequest.prototype.GetXmlElementName = function () { throw new Error("UnsubscribeRequest.ts - GetXmlElementName : Not implemented."); };
    UnsubscribeRequest.prototype.Validate = function () { throw new Error("UnsubscribeRequest.ts - Validate : Not implemented."); };
    UnsubscribeRequest.prototype.WriteElementsToXml = function (writer) { throw new Error("UnsubscribeRequest.ts - WriteElementsToXml : Not implemented."); };
    return UnsubscribeRequest;
}(MultiResponseServiceRequest_1.MultiResponseServiceRequest));
exports.UnsubscribeRequest = UnsubscribeRequest;
