// ews-javascript-api/js/Core/Requests/UpdateDelegateRequest.js
"use strict";
var __extends = (this && this.__extends) || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
};
var DelegateManagementRequestBase_1 = require("./DelegateManagementRequestBase");
/**
 * ## *Not Implemented*
 */
var UpdateDelegateRequest = (function (_super) {
    __extends(UpdateDelegateRequest, _super);
    function UpdateDelegateRequest() {
        _super.apply(this, arguments);
    }
    UpdateDelegateRequest.prototype.CreateResponse = function () { throw new Error("UpdateDelegateRequest.ts - CreateResponse : Not implemented."); };
    UpdateDelegateRequest.prototype.GetMinimumRequiredServerVersion = function () { throw new Error("UpdateDelegateRequest.ts - GetMinimumRequiredServerVersion : Not implemented."); };
    UpdateDelegateRequest.prototype.GetResponseXmlElementName = function () { throw new Error("UpdateDelegateRequest.ts - GetResponseXmlElementName : Not implemented."); };
    UpdateDelegateRequest.prototype.GetXmlElementName = function () { throw new Error("UpdateDelegateRequest.ts - GetXmlElementName : Not implemented."); };
    UpdateDelegateRequest.prototype.Validate = function () { throw new Error("UpdateDelegateRequest.ts - Validate : Not implemented."); };
    UpdateDelegateRequest.prototype.WriteElementsToXml = function (writer) { throw new Error("UpdateDelegateRequest.ts - WriteElementsToXml : Not implemented."); };
    return UpdateDelegateRequest;
}(DelegateManagementRequestBase_1.DelegateManagementRequestBase));
exports.UpdateDelegateRequest = UpdateDelegateRequest;
