// ews-javascript-api/js/Core/Requests/UpdateInboxRulesRequest.js
"use strict";
var __extends = (this && this.__extends) || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
};
var SimpleServiceRequestBase_1 = require("./SimpleServiceRequestBase");
/**
 * ## *Not Implemented*
 */
var UpdateInboxRulesRequest = (function (_super) {
    __extends(UpdateInboxRulesRequest, _super);
    function UpdateInboxRulesRequest() {
        _super.apply(this, arguments);
    }
    UpdateInboxRulesRequest.prototype.Execute = function () { throw new Error("UpdateInboxRulesRequest.ts - Execute : Not implemented."); };
    UpdateInboxRulesRequest.prototype.GetMinimumRequiredServerVersion = function () { throw new Error("UpdateInboxRulesRequest.ts - GetMinimumRequiredServerVersion : Not implemented."); };
    UpdateInboxRulesRequest.prototype.GetResponseXmlElementName = function () { throw new Error("UpdateInboxRulesRequest.ts - GetResponseXmlElementName : Not implemented."); };
    UpdateInboxRulesRequest.prototype.GetXmlElementName = function () { throw new Error("UpdateInboxRulesRequest.ts - GetXmlElementName : Not implemented."); };
    UpdateInboxRulesRequest.prototype.ParseResponse = function (reader) { throw new Error("UpdateInboxRulesRequest.ts - ParseResponse : Not implemented."); };
    UpdateInboxRulesRequest.prototype.Validate = function () { throw new Error("UpdateInboxRulesRequest.ts - Validate : Not implemented."); };
    UpdateInboxRulesRequest.prototype.WriteElementsToXml = function (writer) { throw new Error("UpdateInboxRulesRequest.ts - WriteElementsToXml : Not implemented."); };
    return UpdateInboxRulesRequest;
}(SimpleServiceRequestBase_1.SimpleServiceRequestBase));
exports.UpdateInboxRulesRequest = UpdateInboxRulesRequest;
