// ews-javascript-api/js/Misc/IdConversion/AlternateIdBase.js
"use strict";
var AlternateIdBase = (function () {
    function AlternateIdBase() {
    }
    AlternateIdBase.prototype.GetXmlElementName = function () { throw new Error("AlternateIdBase.ts - GetXmlElementName : Not implemented."); };
    AlternateIdBase.prototype.InternalToJson = function (jsonObject) { throw new Error("AlternateIdBase.ts - InternalToJson : Not implemented."); };
    AlternateIdBase.prototype.InternalValidate = function () { throw new Error("AlternateIdBase.ts - InternalValidate : Not implemented."); };
    AlternateIdBase.prototype.LoadAttributesFromJson = function (responseObject) { throw new Error("AlternateIdBase.ts - LoadAttributesFromJson : Not implemented."); };
    AlternateIdBase.prototype.LoadAttributesFromXml = function (reader) { throw new Error("AlternateIdBase.ts - LoadAttributesFromXml : Not implemented."); };
    AlternateIdBase.prototype.WriteAttributesToXml = function (writer) { throw new Error("AlternateIdBase.ts - WriteAttributesToXml : Not implemented."); };
    AlternateIdBase.prototype.WriteToXml = function (writer) { throw new Error("AlternateIdBase.ts - WriteToXml : Not implemented."); };
    return AlternateIdBase;
}());
exports.AlternateIdBase = AlternateIdBase;
//}
