// ews-javascript-api/js/Misc/IdConversion/AlternatePublicFolderItemId.js
"use strict";
var __extends = (this && this.__extends) || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
};
var AlternatePublicFolderId_1 = require("./AlternatePublicFolderId");
var AlternatePublicFolderItemId = (function (_super) {
    __extends(AlternatePublicFolderItemId, _super);
    function AlternatePublicFolderItemId() {
        _super.apply(this, arguments);
    }
    AlternatePublicFolderItemId.prototype.GetXmlElementName = function () { throw new Error("AlternatePublicFolderItemId.ts - GetXmlElementName : Not implemented."); };
    AlternatePublicFolderItemId.prototype.InternalToJson = function (jsonObject) { throw new Error("AlternatePublicFolderItemId.ts - InternalToJson : Not implemented."); };
    AlternatePublicFolderItemId.prototype.LoadAttributesFromJson = function (responseObject) { throw new Error("AlternatePublicFolderItemId.ts - LoadAttributesFromJson : Not implemented."); };
    AlternatePublicFolderItemId.prototype.LoadAttributesFromXml = function (reader) { throw new Error("AlternatePublicFolderItemId.ts - LoadAttributesFromXml : Not implemented."); };
    AlternatePublicFolderItemId.prototype.WriteAttributesToXml = function (writer) { throw new Error("AlternatePublicFolderItemId.ts - WriteAttributesToXml : Not implemented."); };
    return AlternatePublicFolderItemId;
}(AlternatePublicFolderId_1.AlternatePublicFolderId));
exports.AlternatePublicFolderItemId = AlternatePublicFolderItemId;
//}
