// ews-javascript-api/js/Misc/IdConversion/AlternatePublicFolderId.js
"use strict";
var __extends = (this && this.__extends) || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
};
var AlternateIdBase_1 = require("./AlternateIdBase");
var AlternatePublicFolderId = (function (_super) {
    __extends(AlternatePublicFolderId, _super);
    function AlternatePublicFolderId() {
        _super.apply(this, arguments);
    }
    AlternatePublicFolderId.prototype.GetXmlElementName = function () { throw new Error("AlternatePublicFolderId.ts - GetXmlElementName : Not implemented."); };
    AlternatePublicFolderId.prototype.InternalToJson = function (jsonObject) { throw new Error("AlternatePublicFolderId.ts - InternalToJson : Not implemented."); };
    AlternatePublicFolderId.prototype.LoadAttributesFromJson = function (responseObject) { throw new Error("AlternatePublicFolderId.ts - LoadAttributesFromJson : Not implemented."); };
    AlternatePublicFolderId.prototype.LoadAttributesFromXml = function (reader) { throw new Error("AlternatePublicFolderId.ts - LoadAttributesFromXml : Not implemented."); };
    AlternatePublicFolderId.prototype.WriteAttributesToXml = function (writer) { throw new Error("AlternatePublicFolderId.ts - WriteAttributesToXml : Not implemented."); };
    AlternatePublicFolderId.SchemaTypeName = "AlternatePublicFolderIdType";
    return AlternatePublicFolderId;
}(AlternateIdBase_1.AlternateIdBase));
exports.AlternatePublicFolderId = AlternatePublicFolderId;
//}
