// ews-javascript-api/js/Search/Filters/SearchFilter_ContainsSubstring.js
var __extends = this.__extends || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    __.prototype = b.prototype;
    d.prototype = new __();
};
var SearchFilter_PropertyBasedFilter_1 = require("./SearchFilter_PropertyBasedFilter");
var ContainsSubstring = (function (_super) {
    __extends(ContainsSubstring, _super);
    function ContainsSubstring() {
        _super.apply(this, arguments);
    }
    ContainsSubstring.prototype.GetXmlElementName = function () { throw new Error("SearchFilter_ContainsSubstring.ts - GetXmlElementName : Not implemented."); };
    ContainsSubstring.prototype.InternalToJson = function (service) { throw new Error("SearchFilter_ContainsSubstring.ts - InternalToJson : Not implemented."); };
    ContainsSubstring.prototype.InternalValidate = function () { throw new Error("SearchFilter_ContainsSubstring.ts - InternalValidate : Not implemented."); };
    ContainsSubstring.prototype.LoadFromJson = function (jsonProperty, service) { throw new Error("SearchFilter_ContainsSubstring.ts - LoadFromJson : Not implemented."); };
    ContainsSubstring.prototype.ReadAttributesFromXmlJsObject = function (reader) { throw new Error("SearchFilter_ContainsSubstring.ts - ReadAttributesFromXml : Not implemented."); };
    ContainsSubstring.prototype.ReadElementsFromXmlJsObject = function (reader) { throw new Error("SearchFilter_ContainsSubstring.ts - TryReadElementFromXmlJsObject : Not implemented."); };
    ContainsSubstring.prototype.WriteAttributesToXml = function (writer) { throw new Error("SearchFilter_ContainsSubstring.ts - WriteAttributesToXml : Not implemented."); };
    ContainsSubstring.prototype.WriteElementsToXml = function (writer) { throw new Error("SearchFilter_ContainsSubstring.ts - WriteElementsToXml : Not implemented."); };
    return ContainsSubstring;
})(SearchFilter_PropertyBasedFilter_1.PropertyBasedFilter);
exports.ContainsSubstring = ContainsSubstring;
//------------modulename->Microsoft.Exchange.WebServices.Data.SearchFilter------------
