// ews-javascript-api/js/Search/Filters/SearchFilter_ExcludesBitmask.js
var __extends = this.__extends || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    __.prototype = b.prototype;
    d.prototype = new __();
};
var SearchFilter_PropertyBasedFilter_1 = require("./SearchFilter_PropertyBasedFilter");
var ExcludesBitmask = (function (_super) {
    __extends(ExcludesBitmask, _super);
    function ExcludesBitmask() {
        _super.apply(this, arguments);
    }
    ExcludesBitmask.prototype.GetXmlElementName = function () { throw new Error("SearchFilter_ExcludesBitmask.ts - GetXmlElementName : Not implemented."); };
    ExcludesBitmask.prototype.InternalToJson = function (service) { throw new Error("SearchFilter_ExcludesBitmask.ts - InternalToJson : Not implemented."); };
    ExcludesBitmask.prototype.LoadFromJson = function (jsonProperty, service) { throw new Error("SearchFilter_ExcludesBitmask.ts - LoadFromJson : Not implemented."); };
    ExcludesBitmask.prototype.ReadElementsFromXmlJsObject = function (reader) { throw new Error("SearchFilter_ExcludesBitmask.ts - TryReadElementFromXmlJsObject : Not implemented."); };
    ExcludesBitmask.prototype.WriteElementsToXml = function (writer) { throw new Error("SearchFilter_ExcludesBitmask.ts - WriteElementsToXml : Not implemented."); };
    return ExcludesBitmask;
})(SearchFilter_PropertyBasedFilter_1.PropertyBasedFilter);
exports.ExcludesBitmask = ExcludesBitmask;
//------------modulename->Microsoft.Exchange.WebServices.Data.SearchFilter------------
