// ews-javascript-api/js/Search/Filters/SearchFilter_RelationalFilter.js
var __extends = this.__extends || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    __.prototype = b.prototype;
    d.prototype = new __();
};
var SearchFilter_PropertyBasedFilter_1 = require("./SearchFilter_PropertyBasedFilter");
var RelationalFilter = (function (_super) {
    __extends(RelationalFilter, _super);
    function RelationalFilter() {
        _super.apply(this, arguments);
    }
    RelationalFilter.prototype.InternalToJson = function (service) { throw new Error("SearchFilter_RelationalFilter.ts - InternalToJson : Not implemented."); };
    RelationalFilter.prototype.InternalValidate = function () { throw new Error("SearchFilter_RelationalFilter.ts - InternalValidate : Not implemented."); };
    RelationalFilter.prototype.LoadFromJson = function (jsonProperty, service) { throw new Error("SearchFilter_RelationalFilter.ts - LoadFromJson : Not implemented."); };
    RelationalFilter.prototype.ReadElementsFromXmlJsObject = function (reader) { throw new Error("SearchFilter_RelationalFilter.ts - TryReadElementFromXmlJsObject : Not implemented."); };
    RelationalFilter.prototype.WriteElementsToXml = function (writer) { throw new Error("SearchFilter_RelationalFilter.ts - WriteElementsToXml : Not implemented."); };
    return RelationalFilter;
})(SearchFilter_PropertyBasedFilter_1.PropertyBasedFilter);
exports.RelationalFilter = RelationalFilter;
//------------modulename->Microsoft.Exchange.WebServices.Data.SearchFilter------------
