// ews-javascript-api/js/Search/Filters/SearchFilter_Not.js
var __extends = this.__extends || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    __.prototype = b.prototype;
    d.prototype = new __();
};
var SearchFilter_1 = require("./SearchFilter");
var Not = (function (_super) {
    __extends(Not, _super);
    function Not() {
        _super.apply(this, arguments);
    }
    Not.prototype.GetXmlElementName = function () { throw new Error("SearchFilter_Not.ts - GetXmlElementName : Not implemented."); };
    Not.prototype.InternalToJson = function (service) { throw new Error("SearchFilter_Not.ts - InternalToJson : Not implemented."); };
    Not.prototype.InternalValidate = function () { throw new Error("SearchFilter_Not.ts - InternalValidate : Not implemented."); };
    Not.prototype.LoadFromJson = function (jsonProperty, service) { throw new Error("SearchFilter_Not.ts - LoadFromJson : Not implemented."); };
    Not.prototype.SearchFilterChanged = function (complexProperty) { throw new Error("SearchFilter_Not.ts - SearchFilterChanged : Not implemented."); };
    Not.prototype.ReadElementsFromXmlJsObject = function (reader) { throw new Error("SearchFilter_Not.ts - TryReadElementFromXmlJsObject : Not implemented."); };
    Not.prototype.WriteElementsToXml = function (writer) { throw new Error("SearchFilter_Not.ts - WriteElementsToXml : Not implemented."); };
    return Not;
})(SearchFilter_1.SearchFilter);
exports.Not = Not;
//------------modulename->Microsoft.Exchange.WebServices.Data.SearchFilter------------
