// ews-javascript-api/js/Search/Filters/SearchFilter_PropertyBasedFilter.js
var __extends = this.__extends || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    __.prototype = b.prototype;
    d.prototype = new __();
};
var SearchFilter_1 = require("./SearchFilter");
var PropertyBasedFilter = (function (_super) {
    __extends(PropertyBasedFilter, _super);
    function PropertyBasedFilter() {
        _super.apply(this, arguments);
    }
    PropertyBasedFilter.prototype.InternalToJson = function (service) { throw new Error("SearchFilter_PropertyBasedFilter.ts - InternalToJson : Not implemented."); };
    PropertyBasedFilter.prototype.InternalValidate = function () { throw new Error("SearchFilter_PropertyBasedFilter.ts - InternalValidate : Not implemented."); };
    PropertyBasedFilter.prototype.LoadFromJson = function (jsonProperty, service) { throw new Error("SearchFilter_PropertyBasedFilter.ts - LoadFromJson : Not implemented."); };
    PropertyBasedFilter.prototype.ReadElementsFromXmlJsObject = function (reader) { throw new Error("SearchFilter_PropertyBasedFilter.ts - TryReadElementFromXmlJsObject : Not implemented."); };
    PropertyBasedFilter.prototype.WriteElementsToXml = function (writer) { throw new Error("SearchFilter_PropertyBasedFilter.ts - WriteElementsToXml : Not implemented."); };
    return PropertyBasedFilter;
})(SearchFilter_1.SearchFilter);
exports.PropertyBasedFilter = PropertyBasedFilter;
//------------modulename->Microsoft.Exchange.WebServices.Data.SearchFilter------------
