// ews-javascript-api/js/ComplexProperties/DelegateUser.js
"use strict";
var __extends = (this && this.__extends) || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
};
var ComplexProperty_1 = require("./ComplexProperty");
var DelegateUser = (function (_super) {
    __extends(DelegateUser, _super);
    function DelegateUser() {
        _super.apply(this, arguments);
    }
    DelegateUser.prototype.InternalToJson = function (service) { throw new Error("DelegateUser.ts - InternalToJson : Not implemented."); };
    DelegateUser.prototype.InternalValidate = function () { throw new Error("DelegateUser.ts - InternalValidate : Not implemented."); };
    DelegateUser.prototype.LoadFromJson = function (jsonProperty, service) { throw new Error("DelegateUser.ts - LoadFromJson : Not implemented."); };
    DelegateUser.prototype.ReadElementsFromXmlJsObject = function (reader) { throw new Error("DelegateUser.ts - TryReadElementFromXmlJsObject : Not implemented."); };
    DelegateUser.prototype.ValidateAddDelegate = function () { throw new Error("DelegateUser.ts - ValidateAddDelegate : Not implemented."); };
    DelegateUser.prototype.ValidateUpdateDelegate = function () { throw new Error("DelegateUser.ts - ValidateUpdateDelegate : Not implemented."); };
    DelegateUser.prototype.WriteElementsToXml = function (writer) { throw new Error("DelegateUser.ts - WriteElementsToXml : Not implemented."); };
    return DelegateUser;
}(ComplexProperty_1.ComplexProperty));
exports.DelegateUser = DelegateUser;
