// ews-javascript-api/js/ComplexProperties/RulePredicateDateRange.js
"use strict";
var __extends = (this && this.__extends) || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
};
var ComplexProperty_1 = require("./ComplexProperty");
var RulePredicateDateRange = (function (_super) {
    __extends(RulePredicateDateRange, _super);
    function RulePredicateDateRange() {
        _super.apply(this, arguments);
    }
    RulePredicateDateRange.prototype.InternalToJson = function (service) { throw new Error("RulePredicateDateRange.ts - InternalToJson : Not implemented."); };
    RulePredicateDateRange.prototype.InternalValidate = function () { throw new Error("RulePredicateDateRange.ts - InternalValidate : Not implemented."); };
    RulePredicateDateRange.prototype.LoadFromJson = function (jsonProperty, service) { throw new Error("RulePredicateDateRange.ts - LoadFromJson : Not implemented."); };
    RulePredicateDateRange.prototype.ReadElementsFromXmlJsObject = function (reader) { throw new Error("RulePredicateDateRange.ts - TryReadElementFromXmlJsObject : Not implemented."); };
    RulePredicateDateRange.prototype.WriteElementsToXml = function (writer) { throw new Error("RulePredicateDateRange.ts - WriteElementsToXml : Not implemented."); };
    return RulePredicateDateRange;
}(ComplexProperty_1.ComplexProperty));
exports.RulePredicateDateRange = RulePredicateDateRange;
