// ews-javascript-api/js/ComplexProperties/RulePredicateSizeRange.js
"use strict";
var __extends = (this && this.__extends) || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
};
var ComplexProperty_1 = require("./ComplexProperty");
var RulePredicateSizeRange = (function (_super) {
    __extends(RulePredicateSizeRange, _super);
    function RulePredicateSizeRange() {
        _super.apply(this, arguments);
    }
    RulePredicateSizeRange.prototype.InternalToJson = function (service) { throw new Error("RulePredicateSizeRange.ts - InternalToJson : Not implemented."); };
    RulePredicateSizeRange.prototype.InternalValidate = function () { throw new Error("RulePredicateSizeRange.ts - InternalValidate : Not implemented."); };
    RulePredicateSizeRange.prototype.LoadFromJson = function (jsonProperty, service) { throw new Error("RulePredicateSizeRange.ts - LoadFromJson : Not implemented."); };
    RulePredicateSizeRange.prototype.ReadElementsFromXmlJsObject = function (reader) { throw new Error("RulePredicateSizeRange.ts - TryReadElementFromXmlJsObject : Not implemented."); };
    RulePredicateSizeRange.prototype.WriteElementsToXml = function (writer) { throw new Error("RulePredicateSizeRange.ts - WriteElementsToXml : Not implemented."); };
    return RulePredicateSizeRange;
}(ComplexProperty_1.ComplexProperty));
exports.RulePredicateSizeRange = RulePredicateSizeRange;
