// ews-javascript-api/js/ComplexProperties/RuleActions.js
"use strict";
var __extends = (this && this.__extends) || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
};
var ComplexProperty_1 = require("./ComplexProperty");
var RuleActions = (function (_super) {
    __extends(RuleActions, _super);
    function RuleActions() {
        _super.apply(this, arguments);
    }
    RuleActions.prototype.ConvertSMSRecipientsFromEmailAddressCollectionToMobilePhoneCollection = function (emailCollection) { throw new Error("RuleActions.ts - ConvertSMSRecipientsFromEmailAddressCollectionToMobilePhoneCollection : Not implemented."); };
    RuleActions.prototype.ConvertSMSRecipientsFromMobilePhoneCollectionToEmailAddressCollection = function (recipientCollection /*System.Collections.ObjectModel.Collection<MobilePhone>*/) { throw new Error("RuleActions.ts - ConvertSMSRecipientsFromMobilePhoneCollectionToEmailAddressCollection : Not implemented."); };
    RuleActions.prototype.InternalToJson = function (service) { throw new Error("RuleActions.ts - InternalToJson : Not implemented."); };
    RuleActions.prototype.InternalValidate = function () { throw new Error("RuleActions.ts - InternalValidate : Not implemented."); };
    RuleActions.prototype.LoadFromJson = function (jsonProperty, service) { throw new Error("RuleActions.ts - LoadFromJson : Not implemented."); };
    RuleActions.prototype.ReadElementsFromXmlJsObject = function (reader) { throw new Error("RuleActions.ts - TryReadElementFromXmlJsObject : Not implemented."); };
    RuleActions.prototype.WriteElementsToXml = function (writer) { throw new Error("RuleActions.ts - WriteElementsToXml : Not implemented."); };
    return RuleActions;
}(ComplexProperty_1.ComplexProperty));
exports.RuleActions = RuleActions;
