// ews-javascript-api/js/ComplexProperties/Rule.js
"use strict";
var __extends = (this && this.__extends) || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
};
var ComplexProperty_1 = require("./ComplexProperty");
var Rule = (function (_super) {
    __extends(Rule, _super);
    function Rule() {
        _super.apply(this, arguments);
    }
    Rule.prototype.InternalToJson = function (service) { throw new Error("Rule.ts - InternalToJson : Not implemented."); };
    Rule.prototype.InternalValidate = function () { throw new Error("Rule.ts - InternalValidate : Not implemented."); };
    Rule.prototype.LoadFromJson = function (jsonProperty, service) { throw new Error("Rule.ts - LoadFromJson : Not implemented."); };
    Rule.prototype.ReadElementsFromXmlJsObject = function (reader) { throw new Error("Rule.ts - TryReadElementFromXmlJsObject : Not implemented."); };
    Rule.prototype.WriteElementsToXml = function (writer) { throw new Error("Rule.ts - WriteElementsToXml : Not implemented."); };
    return Rule;
}(ComplexProperty_1.ComplexProperty));
exports.Rule = Rule;
