// ews-javascript-api/js/ComplexProperties/SetRuleOperation.js
"use strict";
var __extends = (this && this.__extends) || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
};
var RuleOperation_1 = require("./RuleOperation");
var SetRuleOperation = (function (_super) {
    __extends(SetRuleOperation, _super);
    function SetRuleOperation() {
        _super.apply(this, arguments);
    }
    SetRuleOperation.prototype.InternalToJson = function (service) { throw new Error("SetRuleOperation.ts - InternalToJson : Not implemented."); };
    SetRuleOperation.prototype.InternalValidate = function () { throw new Error("SetRuleOperation.ts - InternalValidate : Not implemented."); };
    SetRuleOperation.prototype.LoadFromJson = function (jsonProperty, service) { throw new Error("SetRuleOperation.ts - LoadFromJson : Not implemented."); };
    SetRuleOperation.prototype.ReadElementsFromXmlJsObject = function (reader) { throw new Error("SetRuleOperation.ts - TryReadElementFromXmlJsObject : Not implemented."); };
    SetRuleOperation.prototype.WriteElementsToXml = function (writer) { throw new Error("SetRuleOperation.ts - WriteElementsToXml : Not implemented."); };
    return SetRuleOperation;
}(RuleOperation_1.RuleOperation));
exports.SetRuleOperation = SetRuleOperation;
//}
